src/v2/components/Billing/components/CancellationNotice/index.tsx

```diff
@@ -18,7 +18,7 @@ export default class CancellationNotice extends PureComponent<
       onReenable,
       customer: { current_period_end_at },
       ...rest
-    } = this.props;
+    } = this.props
```

Comment:
One thing about this semi setting is it messes up the diff; gonna run prettier over the whole src folder so that this doesn't have to muddy up PRs.